docs/UserGuide.md

```diff
@@ -183,6 +191,38 @@ Expected output (fail):
 ```agsl
 Oops! Invalid fosterer index provided, please check again.
 ```
+### Saving changes in a fosterer's details: `save`
+
+Saves changes in details of the fosterer made in the profile page. 
+<div markdown="span" class="alert alert-primary">
+  :exclamation: <b>Important:</b>
+This command is only available while in fosterer profile view page. 
+This command works only in a profile view page, which  after executing a view command.
```

Comment:
this sentence feels like its lacking a smth after 'which', "which appears"...? "which is shown"...?

make a corresponding change in the doc also, thanks!